AnnoyancesFilter/sections/cookies_general.txt

```diff
@@ -648,6 +648,7 @@
 ###cookies_box
 ###cookies_container
 ###cookies_info
+###cookies_infos
```

Comment:
the rule should be domain-specific

Reply:
and this rule leaves a white space at the top
![leftover](https://user-images.githubusercontent.com/18536328/145203726-204af56d-dd20-424d-981c-d4eb7493af89.png)